docs/user/explanation/documentation-structure.rst

```diff
@@ -0,0 +1,43 @@
+How to structure your documentation
+===================================
+
+Structuring your documentation is an important part of the authoring process.
+It's important to think about the reader of the documentation,
+since they are the primary user of documentation.
+
+One of the largest benefits of a good structure is that it removes questions that keep authors from writing documentation.
+Starting with a blank page is often the hardest part of documentation,
+so anything we can do to remove this problem is a win.
```

Comment:
This is super important stuff! I think this could be great for the first section: "Why you need structure"